src/components/atoms/button/style/button.styled.component.tsx

```diff
@@ -7,13 +7,13 @@ export const Styledbutton = styled(Button)<buttonInterface>`
   ${fontLink.url};
   ${fontLink.fontFamily};
   border-radius: 4px;
-  ${props => props.isPrimaryProp? '' : `
+  ${props => props.isprimaryprop === 'true'? '' : `
```

Comment:
i'm not entirely sure but I think the reason why this is throwing the DOM warning you mentioned is because you have to type the props object here. Have you tried doing so?

Reply:
Yes first warning is that they don't accept camelCase props as we can see on this commit, second warning is they suggest that I should pass string or number instead of Boolean values, though it is a warning only the app still compiled and running, though I checked the docs on react it is suggested as a best practice. If we ignore this it can lead a potential security risks to the app once deployed on prod.

Reply:
To be more elaborate here's the 2nd warning after I fix the camelCase warning..

Console Warning / error
![console](https://user-images.githubusercontent.com/90882133/134832378-a06a55d3-5c7c-4855-81ed-d4c61fbe6b76.png)

Though it is said that it not a boolean attribute, i set it as boolean in the interface.
![bool](https://user-images.githubusercontent.com/90882133/134832473-7b32b1ee-f5ed-42bd-82d4-f7dda94554e8.png)

so thats why I applied the reacts suggestion to clear out the warnings and to apply whats is best practice in accordance with the docs

Reply:
@ClydeIanVananaz What happens if you specify the "props" variable here? Something like ${(props: {isPrimary?: boolean;}) => props.isPrimary ?  ...

Reply:
Hmm I will try, I will get back to you for the feedback :)

Reply:
@junrezapico 

I have tried you suggestion but react doesnt accept that line of code.
![2](https://user-images.githubusercontent.com/90882133/134874371-17788165-d103-4325-a729-25bddd13faf1.png)

I Tried also to be more specific by using the interface,
![1](https://user-images.githubusercontent.com/90882133/134874465-729fb4de-aacf-4a3c-87c1-0ffa4f0722d7.png)

But it has same warning / error showed in console.